src/entities/twin/constants.ts

```diff
@@ -0,0 +1,62 @@
+import { AutoFormValueInfo, AutoFormValueType } from "@/components/auto-field";
```

Comment:
To remove this file

This file is duplicate of src/entities/twin/libs/constants.ts